manager/raftpicker/raftpicker.go

```diff
@@ -131,3 +140,8 @@ func (c *ConnSelector) Conn() (*grpc.ClientConn, error) {
 	c.cc = cc
 	return c.cc, nil
 }
+
+// Stop cancels tracking loop for raftpicker.
+func (c *ConnSelector) Stop() {
```

Comment:
Wouldn't that be racy?

I don't think it's very complicated to add `<-c.done` at the end of `Stop` and `defer close(c.done)` in `updateLoop`.